test/module/irohad/multi_sig_transactions/storage_test.cpp

```diff
@@ -9,27 +9,17 @@
 #include "module/irohad/multi_sig_transactions/mst_test_helpers.hpp"
 #include "multi_sig_transactions/storage/mst_storage_impl.hpp"
 
-auto log_ = logger::log("MstStorageTest");
-
 using namespace iroha;
 
-class StorageTestCompleter : public DefaultCompleter {
- public:
-  bool operator()(const DataType &batch, const TimeType &time) const override {
-    return std::all_of(
-        batch->transactions().begin(),
-        batch->transactions().end(),
-        [&time](const auto &tx) { return tx->createdTime() < time; });
-  }
-};
+auto log_ = logger::log("MstStorageTest");
 
 class StorageTest : public testing::Test {
  public:
   StorageTest() : absent_peer_key("absent") {}
 
   void SetUp() override {
-    storage = std::make_shared<MstStorageStateImpl>(
-        std::make_shared<StorageTestCompleter>());
+    completer_ = std::make_shared<DefaultCompleter>(std::chrono::minutes(0));
```

Comment:
I think better to provide test compliter for testing business-logic and make a test for default compliter.